webapi/Options/VectorMemoryWebOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.ComponentModel.DataAnnotations;
+
+namespace SemanticKernel.Service.Options;
+
+/// <summary>
+/// Configuration settings for connecting to a vector memory web service.
+/// </summary>
+public class VectorMemoryWebOptions
+{
+    /// <summary>
+    /// Gets or sets the endpoint protocol and host (e.g. http://localhost).
+    /// </summary>
+    [Required, Url]
+    public string Host { get; set; } = string.Empty; // TODO update to use System.Uri
```

Comment:
nit: we have started including Issue #s with each TODO we leave ([example](https://github.com/microsoft/chat-copilot/blob/main/webapp/src/components/chat/tabs/PlansTab.tsx#L131C5-L131C5)). can you please add one here?

Reply:
I believe that "TODO" was carried over from the previous QDrant provider or automated generation, I'm not sure what point to make in an issue about it, and I'm happy to remove the comment if there isn't a compelling argument to use Uri.

Reply:
looks like @adrianwyatt left the comment in the previous repo so maybe he can weigh in

Reply:
It is probably fine to either update it to use Uri now or just remove the TODO.